src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -235,11 +202,29 @@ class PaymentMethodList extends Component {
 
     render() {
         return (
-            <FlatList
-                data={this.createPaymentMethodList()}
-                renderItem={this.renderItem}
-                keyExtractor={item => item.key}
-            />
+            <>
+                <FlatList
+                    bounces={false}
```

Comment:
Any reason?

Reply:
The button was requested to be outside of the FlatList https://github.com/Expensify/App/issues/8929#issuecomment-1154362227

Reply:
I meant to ask why are we disabling bounce.

Reply:
To avoid the problem that was initially https://github.com/Expensify/App/issues/8929#issue-1231739326
_The button "Add payment method" is not visible when scrolled up_

Reply:
uhm of course, then I remove `bounces={false}`

Reply:
change made